### ISSUE TITLE
java.lang.ClassNotFoundException: com.sun.java.accessibility.AccessBridge

### ISSUE THREAD

**Issue by Psiae** [Psiae/MLToolBox#1]
https://www.nexusmods.com/manorlords/mods/256?tab=posts&jump_to_comment=147170489
```
﻿Error: exception on EDT thread
java.lang.IllegalStateException: Unable to Install UncaughtExceptionHandler on AWT EDT Thread, cause_msg=Assistive Technology not found: com.sun.java.accessibility.AccessBridge
at dev.psiae.mltoolbox.composeui.c.i.a(SourceFile:1262)
at dev.psiae.mltoolbox.a.a(SourceFile:11)
at MainKt.a(SourceFile:1)
at a.invoke(SourceFile:1000)
at kotlin.coroutines.intrinsics.IntrinsicsKt__IntrinsicsJvmKt$createCoroutineUnintercepted$$inlined$createCoroutineFromSuspendFunction$IntrinsicsKt__IntrinsicsJvmKt$1.invokeSuspend(SourceFile:223)
at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(SourceFile:33)
at kotlin.coroutines.ContinuationKt.startCoroutine(SourceFile:115)
at kotlin.coroutines.jvm.internal.RunSuspendKt.runSuspend(SourceFile:19)
at MainKt.main(SourceFile)
Caused by: java.awt.AWTError: Assistive Technology not found: com.sun.java.accessibility.AccessBridge
at java.desktop/java.awt.Toolkit.newAWTError(Unknown Source)
at java.desktop/java.awt.Toolkit.fallbackToLoadClassForAT(Unknown Source)
at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(Unknown Source)
at java.base/java.util.stream.ReferencePipeline$2$1.accept(Unknown Source)
at java.base/java.util.HashMap$KeySpliterator.forEachRemaining(Unknown Source)
at java.base/java.util.stream.AbstractPipeline.copyInto(Unknown Source)
at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(Unknown Source)
at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(Unknown Source)
at java.base/java.util.stream.AbstractPipeline.evaluate(Unknown Source)
at java.base/java.util.stream.ReferencePipeline.forEach(Unknown Source)
at java.desktop/java.awt.Toolkit.loadAssistiveTechnologies(Unknown Source)
at java.desktop/java.awt.Toolkit.getDefaultToolkit(Unknown Source)
at java.desktop/java.awt.EventQueue.invokeAndWait(Unknown Source)
at java.desktop/javax.swing.SwingUtilities.invokeAndWait(Unknown Source)
at dev.psiae.mltoolbox.composeui.c.i.a(SourceFile:1254)
... 8 more
Caused by: java.lang.ClassNotFoundException: com.sun.java.accessibility.AccessBridge
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(Unknown Source)
at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(Unknown Source)
at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
at java.base/java.lang.Class.forName0(Native Method)
at java.base/java.lang.Class.forName(Unknown Source)
at java.base/java.lang.Class.forName(Unknown Source)
... 23 more
Java(TM) SE Runtime Environment (build 21.0.5+9-LTS-239)
Java HotSpot(TM) 64-Bit Server VM (build 21.0.5+9-LTS-239)
Microsoft Windows [Version 10.0.19045.5131]
```

I assume this is related to: https://github.com/JetBrains/compose-multiplatform/issues/4126
Including `jdk.accessibility` module should fix it

Should use [jdeps](https://docs.oracle.com/en/java/javase/11/tools/jdeps.html) in the future

**Comment by Psiae** [comment]
fix attempt: https://github.com/PsiaeStudio/MLToolBox/commit/0898b72e0e6d297322b2c06a966505c9c38b0326
